gapis/gfxapi/gles/dependency_graph.go

```diff
@@ -418,3 +421,20 @@ func getAttachmentData(g *DependencyGraph, c *Context, att FramebufferAttachment
 	}
 	return
 }
+
+func getAttachmentSize(g *DependencyGraph, c *Context, att FramebufferAttachment) (key stateKey) {
```

Comment:
Nit: I would prefer getAttachmentDataAndSize instead of copy-pasting the method (in the same way it is done with textures).